video_xblock/static/js/student-view/player-state.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * This part is responsible for loading and saving player state.
+ * State includes:
+ * - Current time
+ * - Playback rate
+ * - Volume
+ * - Muted
+ *
+ * State is loaded after VideoJs player is fully initialized.
+ * State is saved at certain events.
+ */
+
+var PlayerState = function(player, playerStateObj) {
+    'use strict';
+    var playerState = {
+        volume: playerStateObj.volume,
+        currentTime: playerStateObj.current_time,
+        playbackRate: playerStateObj.playback_rate,
+        muted: playerStateObj.muted,
+        transcriptsEnabled: playerStateObj.transcripts_enabled,
+        captionsEnabled: playerStateObj.captions_enabled,
+        captionsLanguage: playerStateObj.captions_language
+    };
```

Comment:
it looks weird. Cannot you simply use dict keys in camelCase format to avoid such kind of transformation?
https://github.com/raccoongang/xblock-video/blob/7220e2f159de26b30544904290f56c5a72b69483/video_xblock/video_xblock.py#L305-L313